jenkins-plugin/src/test/java/org/jenkinsci/plugins/pipeline/maven/ConfigurationAsCodeNeedDockerTest.java

```diff
@@ -0,0 +1,74 @@
+package org.jenkinsci.plugins.pipeline.maven;
+
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import hudson.ExtensionList;
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.ConfiguratorRegistry;
+import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.plugins.pipeline.maven.dao.MonitoringPipelineMavenPluginDaoDecorator;
+import org.jenkinsci.plugins.pipeline.maven.dao.PipelineMavenPluginNullDao;
+import org.jenkinsci.plugins.pipeline.maven.db.PipelineMavenPluginMySqlDao;
+import org.junit.Assume;
+import org.junit.jupiter.api.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static io.jenkins.plugins.casc.misc.Util.*;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@WithJenkins
+@Testcontainers(disabledWithoutDocker = true)
+public class ConfigurationAsCodeNeedDockerTest {
+
+    @Container
+    public static MySQLContainer<?> MYSQL_DB = new MySQLContainer<>(MySQLContainer.NAME).withUsername("aUser").withPassword("aPass");
+
+    @Test
+    public void should_support_mysql_configuration(JenkinsRule r) throws Exception {
```

Comment:
could you please add a test for Postgres as well
could be inspired by
https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/jenkins-plugin/src/test/java/org/jenkinsci/plugins/pipeline/maven/GlobalPipelineMavenConfigTest.java#L127

need to change those also:
https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/jenkins-plugin/src/test/resources/org/jenkinsci/plugins/pipeline/maven/configuration-as-code_postgresql.yml
https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/jenkins-plugin/src/test/resources/org/jenkinsci/plugins/pipeline/maven/expected_output_postgresql.yml

Reply:
Postgres test here 7e9a613117d581da151c44c150c16bb70ced8b65